Sources/PointFree/Auth.swift

```diff
@@ -147,7 +180,7 @@ private func githubAuthorizationUrl(withRedirect redirect: String?) -> String {
   let params: [String: String] = [
     "scope": "user:email",
     "client_id": AppEnvironment.current.envVars.gitHub.clientId,
-    "redirect_uri": url(to: .githubCallback(code: "", redirect: redirect))
+    "redirect_uri": url(to: .githubCallback(code: nil, redirect: redirect))
```

Comment:
now we can use `nil` for the redirect and it will exclude the `code` from the query params.